packages/core/src/getters/response.ts

```diff
@@ -37,6 +37,7 @@ export const getResponse = ({
     operationName,
     context,
     'void',
+    (type) => type.key.startsWith('2') + type.value,
```

Comment:
Can `type.key` ever be null?

Reply:
It shouldn't, it comes from `[statuscode: string]` in `ResponsesObject`, which gets picked up from 
```yaml
paths:
  /error:
    post:
      responses:
        '200': 
        # ^
```
So I think this is safe